.gitmodules

```diff
@@ -139,3 +139,6 @@
 [submodule "scripts/esdk-settings-generator"]
 	path = scripts/esdk-settings-generator
 	url = https://github.com/sifive/esdk-settings-generator.git
+[submodule "software/example-freertos-pmp-blinky"]
+	path = software/example-freertos-pmp-blinky
+	url = git@github.com:sifive/example-freertos-pmp-blinky
```

Comment:
Submodule URL need to be in HTTPS format to allow unauthenticated users to clone them.

Reply:
Fixed it.

Reply:
Since this is an example, can you move to where the rest of the examples are. Make it easier to trace later.